api-v2/src/plan/plan.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+import { PlanService } from './plan.service';
+import { PlanController } from './plan.controller';
+import { Plan } from './entities/plan.entity';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([Plan])],
```

Comment:
this is cool ,what does it do? (curious)

Reply:
the repository pattern - informs nest to just make the Plan repository available in providers of the module!